litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -40,6 +40,7 @@ input ClusterInput {
   agent_ns_exists: Boolean
   agent_sa_exists: Boolean
   node_selector: String
+  tolerations: String
```

Comment:
Should we take tolerations as a map or array? It might create issue for users, they will have to escape the tolerations?

Reply:
Okay, changed the schema